README.md

```diff
@@ -1,31 +1,35 @@
-# Fairing
+# Kubeflow Fairing
 
-Easily train ML models on Kubernetes, directly from your python code, or a Jupyter notebook.
+Easily build, train, and deploy your machine learning (ML) training jobs on Kubeflow or Kubernetes,
+directly from Python code or a Jupyter notebook. 
 
 ## Requirements
 
+To use Kubeflow Fairing, you need:
+
 - a kubeconfig
```

Comment:
@r2d4 can we add more context (link to external doc) on what kubeconfig and docker config means?

Reply:
SGTM - opened an issue. Let's merge this first

Reply:
Sounds good.